lib/modules/accountSwitcher.js

```diff
@@ -245,7 +249,9 @@ const switchTo = mutex(async (username: string) => {
 
 		throw e;
 	});
-
+	
+	document.body.style.cursor = oldCursorStyle;
```

Comment:
This might not be executed if any of the above `await ajax` fails.

Reply:
It might be better to add this change here: https://github.com/honestbleeps/Reddit-Enhancement-Suite/pull/5170/files#diff-f0f9061059f0e11ef81314d9f20d428cR200

E.g: 
```
element.addEventListener('click', () => {
  const oldCursorStyle = document.body.style.cursor;
  document.body.style.cursor = 'wait';
  switchTo(username).catch(console.error);
  document.body.style.cursor = oldCursorStyle;
})
```

Reply:
That makes sense, I updated the code to match yours. By the way I really appreciate you walking me through all these changes, I'll keep these coding patterns in mind for the next pull request.